src/calcium/qqbar.jl

```diff
@@ -1552,3 +1540,113 @@ Root 1.41421 of x^2 - 2
 ```
 """
 algebraic_closure(::QQField) = QQBarField()
+
+################################################################################
+#
+#  Auto-generated bindings (DO NOT EDIT)
```

Comment:
I would rather put any kind of autogenerated code in separate files. That makes it much easier to deal with it and helps reduce the risk of accidental edits to generated code.